### ISSUE TITLE
New Colours

### ISSUE THREAD

**Issue by Ademsk1** [nearform/quantum#186]
Nick added in some new colours that we'll need to introduce into the `src/colors` dir

**Comment by nickzafiropulos** [comment]
Wait until https://github.com/nearform/quantum/issues/231 has been done, as colours will be changing.

**Comment by Filippoperl** [comment]
Quantum colors need to be reviewed and adapted to new brand guidelines. Until now the new NF brand guidelines do not have colours for Digital products so we reached out to the brand team to propose to review our colors and create new ones for digital products. Brand team agreed.

To do so 3 dummy pages have been created here--> https://www.figma.com/file/XFbhstkgQFz8ZefAU3w2p4/1.-Quantum-Design-System?type=design&node-id=1889%3A325&mode=design&t=Mzc5slgjOt4vFqV4-1

**The idea is to test our colors**, amend what is not in line with brand guidelines, and add new ones if needed to have the pages with a look and feel more in line with NF brand.

**Comment by simoneb** [comment]
@marceloFerreira90 this is one for you to take a look at when it's a good time to do so

**Comment by gregoryduckworth** [comment]
Picked this up and I think it's been overtaken by events — recommending it be closed as superseded. The evidence:

- The issue names no colour values. The only source for what Nick added is the private Figma file linked in the comments above, so there is nothing in the issue itself that can be implemented.
- Its own first comment says *"Wait until #231 has been done, as colours will be changing."*
- #231 has since been done: PR #344 (`feat: new brand theme`) rewrote `src/colors` and introduced the `brandMidnight` / `brandGreen` ramps, shipping in v1.0.0. It closed #231 but not this one.

So the palette this issue was written against (Nov 2023) is not the palette in `src/colors` today, and any list of colours attached to it predates the rebrand.

If there *are* still colours from Nick that haven't landed, the useful thing would be a fresh issue carrying the actual values — ramp names, stops, hexes, and which semantic tokens (`accent`, `border`, `feedback`, `button.*`) they should feed. Worth knowing for whoever picks that up: `src/colors` is now contrast-gated by `__tests__/contrast.test.ts` (#920), so new ramps want their AA pairings checked as they go in.

One genuine bug did come out of reading the directory, fixed separately in #926: `button.*.selected` spelled its light key `DEFAILT` in all five variants, so `bg-button-primary-selected` resolved to nothing.

**Comment by gregoryduckworth** [comment]
Closing as superseded by #929.

This issue never carried the colour values — the body is one sentence and the only source was a private Figma file from November 2023. Its own first comment said to wait for #231, and #231 was closed on 2024-02-07 by #344 (`feat: new brand theme`, `5ec4760`), which rewrote `src/colors` and introduced the `brandMidnight` / `brandGreen` ramps. The palette this was written against no longer exists, so there is nothing here that can be implemented as written.

#929 replaces it: same intent, with an explicit list of what's needed to make it actionable (ramp names, stops, hexes, semantic wiring, light/dark halves) and notes on the contrast gate in `__tests__/contrast.test.ts` that new ramps have to clear.

If the colours from Nick are still wanted, please add the values to #929.
